utils/src/main/scala/za/co/absa/enceladus/utils/types/Defaults.scala

```diff
@@ -59,21 +65,22 @@ object Defaults {
       case _: DoubleType    => value.toDouble
       case _: LongType      => value.toLong
       case _: StringType    => value
-      case _: DateType      => EnceladusDateParser(dtPattern.getOrElse(throw new IllegalArgumentException("No date format specified."))).parseDate(value)
-      case _: TimestampType => EnceladusDateParser(dtPattern.getOrElse(throw new IllegalArgumentException("No timestamp format specified."))).parseTimestamp(value)
+      case _: DateType      => getParser(dtPattern,"No date format specified.").parseDate(value)
```

Comment:
`getParser()` seems like a vary generic name. Consider using more specific name, like `getDateTimeParser()`

Reply:
Changing to shorter but less generic `getDTParser`